src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_ReadPosRankSumTest.java

```diff
@@ -37,9 +38,10 @@
  * </ul>
  *
  */
-//FIXME - this whole class is essentially code duplicated from ReadPosRankSumTest
 public class AS_ReadPosRankSumTest extends AS_RankSumTest implements AS_StandardAnnotation {
 
+    final ReadPosRankSumTest readPosRankSumTest = new ReadPosRankSumTest();
```

Comment:
I really wanted this class to inherit from AS_RankSumTest _and_ ReadPosRankSumTest, so I picked one and copied the code from the other.  This is far more elegant.